### PR TITLE
fix(utils): remove dependency on query-string and dedupe uint8arrays

### PR DESCRIPTION
**Pull request by v1rtl**
## Description

This PR removes usage of query-string in utils and replaces them with a widely available URLSearchParams. It also dedupes download of uint8arrays (currently it pulls 3.1.0 and 3.1.1 etc)

Related: https://github.com/43081j/ecosystem-cleanup/issues/10

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

Through unit tests

## Checklist

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "events": "3.3.0",
     "isomorphic-unfetch": "3.1.0",
     "lodash.isequal": "4.5.0",
-    "uint8arrays": "3.1.0"
+    "uint8arrays": "^5.1.0"
```
Comment (glitch-txs) [inline packages/core/package.json]:
```suggestion
    "uint8arrays": "5.1.0"
```
All versions should be fixed

Comment (v1rtl) [reply packages/core/package.json]: my bad, just fixed them now

File: packages/utils/package.json [hunk 2]
```diff
@@ -43,7 +43,8 @@
     "@walletconnect/window-getters": "1.0.1",
     "@walletconnect/window-metadata": "1.0.1",
     "detect-browser": "5.3.0",
-    "uint8arrays": "5.1.0"
+    "uint8arrays": "5.1.0",
+    "elliptic": "^6.5.7"
```
Comment (glitch-txs) [inline packages/utils/package.json]:
```suggestion
    "elliptic": "6.5.7"
```

Fixed versioning + update package-lock.json 🙏

**Comment by v1rtl** [comment]
@ganchoradkov besides the lockfile, is there anything else that needs to be fixed?

**Comment by v1rtl** [comment]
closing due to the PR being ignored

**Comment by ganchoradkov** [comment]
hey @talentlessguy, sorry for the delay in merging it, the PR just slipped through the cracks. If you could reopen the PR please we would be more than happy to merge 🙏🙏🙏🙏

**Comment by v1rtl** [comment]
I'll re open two separate ones — one with deduping and another with with replacing query-string so it's easier to review and I don't have to fight merge conflicts

**Pull request closed without merging**